scripts/actions/spells/black/meteor.lua

```diff
@@ -26,6 +26,12 @@ spellObject.onSpellCast = function(caster, target, spell)
     local dmg = 0
     if caster:isPC() then
         dmg = ((100 + caster:getMod(xi.mod.MATT)) / (100 + target:getMod(xi.mod.MDEF))) * (caster:getStat(xi.mod.INT) + caster:getSkillLevel(xi.skill.ELEMENTAL_MAGIC) / 6) * 3.5
+    elseif -- Behemoth family
+        caster:isMob() and
+        (caster:getFamily() == 51 or
+        caster:getFamily() == 479)
+    then
+        dmg = 14 + caster:getMainLvl() * 30
```

Comment:
The mob check is unneeded here. we already checked that the caster isnt a pc, so its a mob.

Can you link to something for this numbers?

Reply:
Jimmayus's monster stats: https://docs.google.com/spreadsheets/d/1YBoveP-weMdidrirY-vPDzHyxbEI2ryECINlfCnFkLI

Under the behemoth mob family on the Monster ftps tab